### ISSUE TITLE
Could not load library cudnn_ops_infer64_8.dll. Error code 126

### ISSUE THREAD

**Issue by KagamiMiro** [pytorch/pytorch#46312]
After upgrade to rtx3090, I reinstalled my Windows10 and pytorch with

>conda install pytorch torchvision cudatoolkit=11 -c pytorch-nightly

When I use nn.Conv/nn.BatchNorm layer in my module and do the forward propagation, it shows

>Could not load library cudnn_ops_infer64_8.dll. Error code 126
>Please make sure cudnn_ops_infer64_8.dll is in your library path!

in Vscode

>Process finished with exit code -1073740791 (0xC0000409)

in Pycharm

I’ve tried reinstalling pytorch and installed cuda toolkit 11 from developer.nvidia.com, but the problem remains unsolved.

```
import torch
import torch.nn as nn

class test(nn.Module):
    def __init__(self):
        super(test, self).__init__()
        self.l = nn.Conv2d(1,1,3)
    def forward(self, input):
        return self.l(input)

net = test().cuda()
print(net(torch.rand(1,1,5,5).cuda()))
```

cc @malfet @seemethere @walterddr @peterjc123 @maxluk @nbcsm @guyang3532 @gunandrose4u @smartcat2010 @mszhanyi

**Comment by malfet** [comment]
I think we just missed this dll from our packaging scripts. Thank you for reporting

**Comment by malfet** [comment]
According to https://app.circleci.com/pipelines/github/pytorch/pytorch/226808/workflows/ca8f2f51-ec15-468d-a8d2-7c3217418348/jobs/8209403
the libraries is copied into the nightly .whl package:
```
  adding 'torch/lib/cudnn_ops_infer64_8.dll'
```

**Comment by zhuqiangLu** [comment]
Hi, @KagamiMiro 

You can download the cudnn achieve from https://developer.nvidia.com/rdp/cudnn-archive and simply put the dlls in your pytorch lib. For example, in my case, I need to put these dlls to  E:\ANACONDA\Lib\site-packages\torch\lib

**Comment by peterjc123** [comment]
Thanks for.reporting. I will take a look.

**Comment by peterjc123** [comment]
Looks like the anaconda team forget to package the `cudnn_ops_*.dll` into the package. Should we package them in our package, @malfet ? Or should we open an issue at somewhere, cc @soumith ?

**Comment by KagamiMiro** [comment]
Thank you guys so much!

**Comment by peterjc123** [comment]
Verified in 1.8.0.dev20201016 builds.

**Comment by KagamiMiro** [comment]
> Verified the fix [pytorch/builder#557](https://github.com/pytorch/builder/pull/557) is working in 1.8.0.dev20201016 builds.

 Perfectly fixed👌

**Comment by surendrallam** [comment]
Hi,
Have faced the same issue, could able to fix it by downloading the files from CUDNN and copy to "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDNN\v5.1\bin".
